Fix Manufacturer Specific Data With Only Company ID

toddwilson opened the pull request:
Fix bug on iOS where devices that only have the company ID but no
other manufacturer specific data have the key omitted in
AdvertisementData.manufacturerData.

toddwilson committed:
Fix Manufacturer Specific Data With Only Company ID

Fix bug on iOS where devices that only have the company ID but no
other manufacturer specific data have the key omitted in
AdvertisementData.manufacturerData.

toddwilson committed:
Fix Manufacturer Specific Data With Only Company ID

Fix bug on iOS where devices that only have the company ID but no
other manufacturer specific data have the key omitted in
AdvertisementData.manufacturerData.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/pauldemarco/flutter_blue?pullRequest=293) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/pauldemarco/flutter_blue?pullRequest=293) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/pauldemarco/flutter_blue?pullRequest=293) it.</sub>

lidongze91 committed:
add minimum sdk version

00Zion00 committed:
modify Android connect device method (#229)

mehmetf committed:
Merge pull request #302 from lidongze91/addMinSDK

Add minimum sdk version (same as what Flutter itself supports)

lidongze91 committed:
update the minimum sdk version and changelog (#329)

* update the minimum sdk version and changelog

* update pubspec.yaml

pauldemarco committed:
Do not append duplicate gatt server when reconnecting. Fix #317.

pauldemarco committed:
Bump version to 0.6.0+3

pauldemarco committed:
Force gattServer.close if device is already disconnected. Fix #317

pauldemarco committed:
Bump version for publishing

pauldemarco committed:
MTU (Maximum Transmission Unit) (#363)

* Update protobuf

* Add MtuSizeRequest MtuSizeResponse proto messages

* Add BluetoothDeviceCache. Implement requestMtu and mtu. (Android)

The BluetoothDeviceCache class is used to augment the underlying
Bluetooth API cache for missing information, such as mtu size.

* Add quick ListTile to test MTU features.

* Add requestMtu and mtu to BluetoothDevice.

* Doc updates. Run dartfmt on example.

* Update readme.

* Add mtu and requestMtu to iOS.

* Add comment about which type to use for iOS mtu size.

* Update generated iOS project file.

pauldemarco committed:
Update iOS project files.

pauldemarco committed:
Update CHANGELOG and pubspec for new version 0.6.1

pauldemarco committed:
Doc updates, bump to 0.6.2.

pauldemarco committed:
Report MTU size on didConnectPeripheral/didDiscoverServices, fixes #369.

kschulz-samsung committed:
Better documentation for requestMTU on iOS (#396)

requestMTU is not supported on iOS so this needs better documentations.
issue: https://github.com/pauldemarco/flutter_blue/issues/395

pauldemarco committed:
Update project files (#401)

* Update project files for Android and iOS
* Remove dependency on protoc in iOS podspec file

pauldemarco committed:
Bump version to 0.6.3

hanabi1224 committed:
bump up protobuf major version (#392)

pauldemarco committed:
Write random bytes in example app.

pauldemarco committed:
Various readme updates.

pauldemarco committed:
Add GPB_USE_PROTOBUF_FRAMEWORK_IMPORTS=1 to podspec

pauldemarco committed:
Fix warning about casting in NSLog

pauldemarco committed:
Bump version to 0.6.3+1

pauldemarco committed:
Add discord chat badge to README.

toddwilson committed:
Merge branch 'fix_ios_manufacturer_data' of https://github.com/toddwilson/flutter_blue into fix_ios_manufacturer_data
